### ISSUE TITLE
go.vetFlags does not support ${workspaceRoot}

### ISSUE THREAD

**Issue by tamayika** [microsoft/vscode-go#2527]
I'm using custom go analyzer using https://godoc.org/golang.org/x/tools/go/analysis and https://godoc.org/golang.org/x/tools/go/analysis/unitchecker.
In command line, I can execute like below.

```sh
$ go vet -vettool=`which mytool`
```

I also want to use mytool in vscode-go extension, but vscode runs `go vet`  in package dir, so I must set absolute path like below.

```json
{
  "go.vetFlags": [
    "-vettool=${workspaceRoot}/bin/mytool"
  ]
}
```
However "go.vetFlags" does not support `${workspaceRoot}`, it remains as is.

I found `go.lintFlags` support `${workspaceRoot}` especially for `config` flag.
https://github.com/microsoft/vscode-go/blob/af6f7a0497f2bd99255cbf8868f5f940369f6f08/src/goLint.ts#L70-L85

Is it difficult to support this feature in vscode-go?

**Comment by ramya-rao-a** [comment]
Its not difficult

You see the support for $workspaceRoot for just the config flag in the lint flags. The same can be done for `-vettool` as well.

PRs to add this feature are most welcome.
Code Pointers:
- [Build and Debug the extension](https://github.com/Microsoft/vscode-go/wiki/Building,-Debugging-and-Sideloading-the-extension-in-Visual-Studio-Code#building-and-debugging-the-extension)
- goVet.ts is the file to update and https://github.com/microsoft/vscode-go/blob/af6f7a0497f2bd99255cbf8868f5f940369f6f08/src/goLint.ts#L70-L85 can be taken as reference

**Comment by tamayika** [comment]
thanks for the reply. I'll make a PR

**Comment by ramya-rao-a** [comment]
Your changes are in the [latest beta version](https://github.com/microsoft/vscode-go/wiki/Use-the-beta-version-of-the-latest-Go-extension) of this extension. Can you try it out and confirm that the fix works as expected?

**Comment by tamayika** [comment]
Thanks for the beta release. I confirmed it works as expected!

**Comment by ramya-rao-a** [comment]
This feature is now available in the latest update (0.11.0) to the Go extension. Thanks @tamayika!
